pkg/controlplane/manager.go

```diff
@@ -22,7 +22,8 @@ import (
 )
 
 const (
-	kubeletPort = "10250"
+	kubeletPort    = "10250"
+	etcdHealthPort = "9980"
```

Comment:
Why did you choose this port? Is it a well known port for etcd health?

Reply:
yes, it's taken from [here](https://github.com/openshift/cluster-etcd-operator/blob/master/bindata/etcd/pod.yaml#L205-L209)